ci/infra/aws/iam_policies.tf

```diff
@@ -2,15 +2,43 @@
 # The expected policies are defined here: https://github.com/kubernetes/cloud-provider-aws#readme
 
 locals {
-  aws_iam_policy_master_terraform = "${var.stack_name}_cpi_master_policy"
-  aws_iam_policy_worker_terraform = "${var.stack_name}_cpi_worker_policy"
+  aws_iam_role_policy_master_terraform = "${var.stack_name}_cpi_master_policy"
+  aws_iam_role_policy_worker_terraform = "${var.stack_name}_cpi_worker_policy"
```

Comment:
aws_iam_instance_profile_worker_terraform = "${var.stack_name}_cpi_worker"

Reply:
changed and tested.